### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by Neo0214** [Red-Reality/palm_recognition#1]
Hi bro I really appreciate your work.

**Comment by Red-Reality** [comment]
This work doesn't run as expected. I will be really appreciated if you could help us improve our palm matching model

**Comment by Neo0214** [comment]
Of course. What can I do for you? I'd like to give you a discounted price.
